pmpsui/preemptive_requests.py

```diff
@@ -453,6 +478,15 @@ def update_jf_trans_for_label(self, label: QtWidgets.QLabel) -> None:
             scaled = raw_trans
         label.setText(f"{scaled:.2e}")
 
+    def update_veto(self, value, byte):
+        # nonzero values correspond to veto by the device corresponding to that index
+        veto_devices = [str(device) for device, vetoing in enumerate(value) if vetoing]
+        # update the color manually since we aren't using a channel
+        is_vetoed = len(veto_devices) > 0
+        byte._indicators[0].setColor(byte._on_color if is_vetoed else byte._off_color)
+        # update the tooltip
+        byte.PyDMToolTip = ', '.join(veto_devices)
```

Comment:
In poking around I wasn't able to see these tooltips.  I suppose this function only fires if the veto status changes?  Is it worth running this once at connection?  Maybe also supplying this to `PyDMChannel(connection_slot=...)` might do the trick?

Reply:
I believe I wasn't able to see tooltips on any of the byte indicators, regardless of whether it was veto'd or not.  (I've confirmed this is the case)

Reply:
I also dumped a few print statements into that callback and never saw it fire, for whatever that's worth

Reply:
So sorry I had my dev environment configured incorrectly.  I can see the callbacks fire on startup, but I suppose there are no veto'd devices for tooltips to be populated to.

I think as you mentioned in the description, it's difficult to fully test this without veto-ing a device.  I did invert the conditional on `veto_devices` to confirm the ByteIndicator would light up and the tooltip would populate.  This all worked but the device names were also just integers (produced by `enumerate`).  I'm not sure how useful that information is